Pod/Classes/MentionComposer.swift

```diff
@@ -262,14 +264,26 @@ public class MentionComposer<T: UIView where T: ComposableAttributedTextContaini
 
         // Check for a mention query
         if let query = mentionQuery(fromString: text) {
+            requestingMentionUsers = true
+            refreshTableView()
```

Comment:
Sucks that we can't really effectively monitor changes within the userNameMatches array, otherwise we could just refresh the table view implicitly.